CriticalMass/Colors.swift

```diff
@@ -9,11 +9,7 @@ import UIKit
 
 extension UIColor {
     private class func safeInit(red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) -> UIColor {
-        if #available(iOS 10.0, *) {
-            return UIColor(displayP3Red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
-        } else {
-            return UIColor(red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
-        }
+        return UIColor(displayP3Red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
```

Comment:
Minor: this helper is no longer needed

Reply:
True. Altough I liked the shorter init I removed it to not add another convenience initializer and using one that also Zeplin would produce